src/wallpaperController.ts

```diff
@@ -560,6 +561,9 @@ class WallpaperController {
             this._historyController.insert(newImageEntries.reverse());
 
             this._runPostCommands();
+
+            if (this._settings.getBoolean('show-notifications'))
+                Notification.newWallpaper(newImageEntries.reverse());
```

Comment:
I don't think reversing matters here. The notification only shows the sources, and it would require opening each history info element in the panel menu to recognize the order.

I reversed the order in the lines above so the leftmost new wallpaper is on the top of the history list. Reflecting the usual reading direction.

Reply:
While the notification isn't super useful, I think the order of information in the notification and the history should be consistent.